package.json

```diff
@@ -32,7 +32,9 @@
     "url": "https://github.com/gnosis/dex-contracts/issues"
   },
   "homepage": "https://github.com/gnosis/dex-contracts#readme",
-  "dependencies": {},
+  "dependencies": {
+    "openzeppelin-solidity": "2.1.1"
```

Comment:
I am not sure here, 

open zeppelin seems to be a big dependency. Can't we just import the things that we need here?

Reply:
I wasn't aware that this is possible, but I can investigate.

Reply:
The reason I am using it this way is partially due to the fact that we need the specific release of open-zeppelin for Solidity 0.5. I was essentially following the guide as proposed here in their release guide

[OZ-releases](https://github.com/OpenZeppelin/openzeppelin-solidity/releases)